Foundation/NSData.swift

```diff
@@ -930,7 +930,11 @@ open class NSMutableData : NSData {
     }
     
     open func resetBytes(in range: NSRange) {
+        #if os(Android)
+        memset(mutableBytes.advanced(by: range.location), 0, range.length)
+        #else
         bzero(mutableBytes.advanced(by: range.location), range.length)
+        #endif
```

Comment:
Why not change all uses of `bzero` to `memset`?  `bzero` is considered deprecated, and POSIX1-2008 removes the specification of `bzero` entirely.